linux/lib/mprisService.js

```diff
@@ -96,36 +98,76 @@ module.exports = (win, player, app) => {
     }
   });
 
+  mprisPlayer.on('seek', (seek) => {
+    if (mprisPlayer.playbackStatus !== 'Stopped') {
+      if (seek < 0) seek = ~seek;
```

Comment:
@acrisci this is the only issue we have with `mpris-service` right now. I already reported here: https://github.com/emersion/mpris-service/issues/27
We're waiting for the fix to come so I can remove this workaround and probably ship the new version

Reply:
Just waiting on people to approve some other implementations before the release comes with this fix. Should be ready sometime in the next week.